tests/Unit/Domain.LinnApps.Tests/RequisitionHeaderTests/WhenCreatingAndMissingNominal.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Linn.Stores2.Domain.LinnApps.Tests.RequisitionHeaderTests
+{
+    using Linn.Stores2.Domain.LinnApps.Exceptions;
```

Comment:
I think these should be in alphaetical order to keep stylecop or resharper or whatever it is happy

Reply:
don't know why but mine isn't as fussy as everyone elses